README.md

```diff
@@ -350,6 +350,36 @@ Future.reject('it broke')
 //> Left('it broke')
 ```
 
+#### finally
+##### `#finally :: Future a b ~> Future a c -> Future a b`
+##### `.finally :: Future a b -> Future a c -> Future a c`
```

Comment:
I like that idea! Having the return types match would make it clear that the arguments are flipped (and that there's no typo).